fdbserver/ClusterController.actor.cpp

```diff
@@ -4414,6 +4459,11 @@ ACTOR Future<Void> updatedChangedDatacenters(ClusterControllerData* self) {
 
 ACTOR Future<Void> updateDatacenterVersionDifference(ClusterControllerData* self) {
 	state double lastLogTime = 0;
+
+	// The purpose of the initial delay is to wait for the cluster to achieve a steady state before calculating the DC
+	// version difference, since DC version difference may trigger a failover, and we don't want that to happen too
+	// frequently.
+	wait(delay(SERVER_KNOBS->INITIAL_UPDATE_CROSS_DC_INFO_DELAY));
```

Comment:
What do you think moving this `grace period` checking to the failover logic, so that the version difference is checked anyway, but it is up to the CC to decide whether to use that information?
I don't have a strong preference on this.